it/test/connectors/CalculationListConnectorISpec.scala

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import helpers.{ComponentSpecBase, WiremockHelper}
+import models.calculationList.{CalculationListModel, CalculationListResponseModel}
+import models.errors.{InvalidJsonResponse, UnexpectedJsonFormat, UnexpectedResponse}
+import play.api.http.Status._
+import play.api.libs.json.{JsValue, Json}
+
+class CalculationListConnectorISpec extends ComponentSpecBase {
+
+  lazy val connector: CalculationListConnector = app.injector.instanceOf[CalculationListConnector]
```

Comment:
not sure if lazy evaluation is making any difference here

Reply:
Will update it now